components/brave_wallet/browser/internal/cardano_tx_decoder.rs

```diff
@@ -78,33 +82,43 @@ macro_rules! impl_error {
 }
 
 /// CXX bridge module for FFI with C++
-#[cxx::bridge(namespace = brave_wallet)]
+#[cxx::bridge(namespace = brave_wallet::cardano_rust)]
 mod ffi {
     #[derive(Clone)]
-    struct CxxRestoredCardanoInput {
-        tx_hash: Vec<u8>,
+    struct SerializableTxInput {
+        tx_hash: [u8; 32],
```

Comment:
this is not part of renaming so should be a separate PR along with any other functional changes